code-challenges.test.js

```diff
@@ -62,15 +111,38 @@ var middleLetters1 = "alpha"
 
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
+var middleLetters3 = "code"
+//Expected output: "od"
 
-
-
+describe("when 'middleLetterChecker' is called", () => {
+  let middleLetter1Result = "p"
+  let middleLetter2Result = "oc"
+  let middleLetter3Result = "od"
+  it("returns only the middle letters of the string provided", () => {
+    expect(middleLetterChecker(middleLetters1)).toEqual(middleLetter1Result)
+    expect(middleLetterChecker(middleLetters2)).toEqual(middleLetter2Result)
+    expect(middleLetterChecker(middleLetters3)).toEqual(middleLetter3Result)
+  })
+})
 
 
 // b) Create the function that makes the test pass.
-
-
-
+//write a function that takes a string of single words as an arguement
+//for each - check length of string.  divide number by 2 to find what index is middle
+//round number to ceiling and return letter from that index
+//if even -- return index + 1
+
+const middleLetterChecker = (string) => {
+  let stringLength = string.length
+  if(stringLength % 2 !== 0){
+    return string.charAt(stringLength / 2)
```

Comment:
consider storing `string.charAt(stringLength /2)` into a variable since it is used multiple times